vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/uphold/get_capabilities/get_capabilities.cc

```diff
@@ -69,9 +72,9 @@ GetCapabilities::ProcessResponse(const type::UrlResponse& response) {
           std::move(capability_map)};
 }
 
-std::map<std::string, bool> GetCapabilities::ParseBody(
+std::map<std::string, std::pair<bool, bool>> GetCapabilities::ParseBody(
     const std::string& body) {
-  std::map<std::string, bool> capability_map;
+  std::map<std::string, std::pair<bool, bool>> capability_map;
```

Comment:
Is the `pair` here necessary? Can the existing `bool` just be `false` if the requirements fields is not empty?

If we need both values, then it might be a good idea to introduce a struct here to make things more self-documenting (in a future PR would be fine).

Reply:
I'd prefer staying in the "Uphold realm" and not deriving new data in `ParseBody()`, but I did introduce a struct.